### DISCUSSION TITLE
How can I add a template prompt to a OpenAIFunction Agent?

### DISCUSSION THREAD

**Discussion by remixdance** [FlowiseAI/Flowise#872] [Q&A]
Hi created this flow, and i want to add a prompt template to answers all the user question in a certain way, what is the best option to do that? or how can I link this flow whit a Template prompt 

![image](https://github.com/FlowiseAI/Flowise/assets/42038961/7a30edc0-d5d8-451c-9950-ad6ce8a18ac0)

**Comment by HenryHengZJ** [comment]
You can't use PrompTemplate with an agent. Another way you can do is specify the system message of OpenAI Function Agent
